src/domain/access/role-set/role.set.resolver.mutations.membership.ts

```diff
@@ -634,6 +638,11 @@ export class RoleSetResolverMutationsMembership {
       const invitation =
         await this.roleSetService.createInvitationExistingContributor(input);
 
+      await this.invitationAuthorizationService.applyAuthorizationPolicy(
+        invitation,
+        roleSet.authorization
+      );
```

Comment:
I don't believe that will pick up the latest roleSet authorization reset on line 310 and it might be working for you because there was no change to it - can you please double check.